source/fundamentals/collations.txt

```diff
@@ -4,5 +4,298 @@ Collations
 
 .. default-domain:: mongodb
 
-A collation is a set of character ordering and matching rules that
-apply to a specific language and locale.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use **collations** to run
+operations using string ordering for specific languages and locales (a
+community or region that shares common language idioms).
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following French words: **cote**, **coté**, **côte**, and **côté**.
+
+MongoDB sorts them in the following order using the default binary collation:
+
+.. code-block:: none
+
+   cote
+   coté
+   côte
+   côté
+
+MongoDB sorts them in the following order using the Canadian French collation:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Usage
+-----
+
+You can specify a collation when you create a new collection, view, or index. You can also
+define a collation for specific operations that support collation. 
```

Comment:
Issue: I believe "support collation" in this sentence means "support collecting and combining". A believe this should say "support specifying a collation. 

Issue: I believe "You can also define a collation for specific operations that support specifying a collation" is a redundant statement. Consider changing to something like this:

```suggestion
You can specify a collation when you create a new collection, view, or index. You can also
specify a collation in a subset of operations. 
```